### PR TITLE
fixes on staking after tests

### PR DESCRIPTION
**Pull request by awojciak**

### CODE REVIEW

File: src/components/Cards/ValueCard.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import React from 'react'
-import { Card, CardContent, Typography, Divider, Tooltip, Icon } from '@material-ui/core'
+import React, { useState } from 'react'
+import { Card, CardContent, Typography, Divider, Tooltip, Icon, ClickAwayListener, Hidden } from '@material-ui/core'
 import useStyles from './style'
```
Comment (wojciech-cichocki) [inline src/components/Cards/ValueCard.tsx:3]: `useStyles` should be the last import

File: src/components/WrappedActionMenu/ActionTemplate/style.ts [hunk 2]
```diff
@@ -3,8 +3,10 @@ import { colors } from '@static/theme'
 
 const useStyles = makeStyles((theme: Theme) => ({
   root: {
+    marginTop: 8,
     minHeight: 200,
```
Comment (wojciech-cichocki) [inline src/components/WrappedActionMenu/ActionTemplate/style.ts:7]: now rewards tab are slightly longer than other tabs

**Review by wojciech-cichocki — CHANGES_REQUESTED** [review]

File: src/components/Cards/ValueCard.tsx (hunk 1 above)
Comment (awojciak) [reply src/components/Cards/ValueCard.tsx]: przeniesione

File: src/components/WrappedActionMenu/ActionTemplate/style.ts (hunk 2 above)
Comment (awojciak) [reply src/components/WrappedActionMenu/ActionTemplate/style.ts]: już naprawione

**Review by wojciech-cichocki — APPROVED** [review]

**Pull request merged**